desktop/install/linux-install.md

```diff
@@ -268,6 +270,19 @@ Docker Desktop for Linux runs a Virtual Machine (VM) for the following reasons:
 
     As such, we have adjusted the default memory available to the VM in DD4L. You can tweak this setting to your specific needs by using the **Memory** slider within the **Settings** > **Resources** tab of Docker Desktop.
 
+## File sharing
+
+Docker Desktop for Linux uses [virtiofs](https://virtio-fs.gitlab.io/) as the default (and currently only) mechanism to enable file sharing between the host and Docker Desktop VM. In order not to require elevated privileges, without unnecessarily restricting operations on the shared files, Docker Desktop runs the file sharing service (`virtiofsd`) inside a [user namespace](https://man7.org/linux/man-pages/man7/user_namespaces.7.html) with UID and GID mapping configured. As a result Docker Desktop will rely on the host to be configured to enable the current user to use subordinate ID delegation. For this to be true [/etc/subuid](https://man7.org/linux/man-pages/man5/subuid.5.html) and [/etc/subgid](https://man7.org/linux/man-pages/man5/subgid.5.html) have to be present (Docker Desktop currently only supports subordinate ID delegation configured via files). In case of both Docker Desktop will use the first entry corresponding to the current user, to set up mappings for IDs above 0, and map the current users ID and GID to 0.
```

Comment:
Don't have time right now to do a full review, but some quick blurbs that may help other reviewers;


- perhaps we should refer to the man pages using (e.g.) `subuid(5)`; something like "see `subuid(5)`"
- for the external links, we may want to add `{:target="_blank" rel="noopener"}` to open in a new tab/window
- if possible, try to hard-wrap these paragraphs to (~) 80-chars; it makes reviewing easier (point to a specific passage), and we _try_ to stick with that in most parts
- avoid future tense


Generally, future tense should be avoided as readers read these docs as "do this, and docker does this". Adding future tense adds some cognitive overload ("if I do this, then docker will (eventually? when?) do this"

e.g.;

> As a result Docker Desktop will rely on the host

Can be (perhaps "as a result" can be skipped as well to make it more to the point);

> Docker Desktop relies on the host ..

Similar to "currently" (which may lead to the question: "are changes planned? will this be added?"); the docs
reflect the current state of things, so we can avoid some of those, e.g.;

> (Docker Desktop currently only supports subordinate ID delegation configured via files) In case of both Docker Desktop will use the first entry corresponding to the current user, to set up mappings for IDs above 0, and map the current users ID and GID to 0.

could be;

> Docker Desktop only supports subordinate ID delegation configured via files. In case of both Docker Desktop uses the first entry corresponding to the current user to set up mappings for IDs above 0, and map the current users ID and GID to 0.

The second sentence is a bit confusing (will have to read it a couple of times I think);

- "In case of both" (both what?)
- "the first  entry ... -> to set up mappings for IDs above 0"

Wondering if we need a simple example or a table to just show it, e.g.

| User ID in container | User ID on host     |
|----------------------|---------------------|
| 0 (root)             | 100000 (100000 + 0) |
| 1                    | 100001 (100000 + 1) |
| etc..                |                     |

Reply:
Coincidentally, @tianon was asking me recently about this area; perhaps he has some ideas as well 😅

Reply:
@p1-0tr I notice the external links disappeared in the latest patches, was it intended?

Reply:
Yes, I went with just referring to the relevant man pages (so that users can look them up locally), rather than relying on an external source that stores them.